tests/supervisors/test_multiprocess.py

```diff
@@ -13,9 +18,11 @@ async def app(scope: Scope, receive: ASGIReceiveCallable, send: ASGISendCallable
 
 
 def run(sockets: list[socket.socket] | None) -> None:
-    pass  # pragma: no cover
+    while True:
+        time.sleep(1)
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="In Windows, Ctrl+C/Ctrl+Break will sent to the parent process.")
```

Comment:
Can someone help me solve this problem? Or should I ignore the coverage here?

Reply:
https://github.com/abersheeran/zibai/blob/0d962e0979ba26df809da030469fb7bc2500ce12/tests/test_multiprocess.py#L19 

I am confident that I have found the solution. The Qingming Festival holiday in China is approaching, and I will complete it.